### PR TITLE
Closes #1548: Clean up existing configuration drift ahead of Drupal 9.3.x / Quickstart 2.3.0.

### LINKED ISSUE #1548: Clean up existing configuration drift ahead of 2.3.0
While working on #1180 and #1537 a lot of config files were found to have existing updates when exporting from a clean install of Quickstart before even adding the Drupal 9.3.x changes.

We should update our config to address this configuration drift before merging the 9.3.x changes.

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
Re-exported all Quickstart-managed configuration files to address existing configuration drift.

Merging this now will make it easier to review the actual Drupal 9.3.x-specifc config changes.

## Related issues

Closes #1548 

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [x] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [ ] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_core/config/quickstart/search.page.node_search.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ dependencies:
   module:
     - node
 id: node_search
```
Comment (trackleft) [inline modules/custom/az_core/config/quickstart/search.page.node_search.yml:6]: Should we delete this file?

**Review by trackleft — APPROVED** [review]
I think this looks fine, should we make a follow-up ticket to delete that old node search file?

**Comment by joeparsons** [comment]
Some links that provide some background on where some of these changes are coming from:

[Deprecate the 'roles' property of text formats - Drupal core issue](https://www.drupal.org/project/drupal/issues/3167198)
[Pathauto adds path field to entities](https://git.drupalcode.org/project/pathauto/-/blob/8.x-1.10/pathauto.module#L121-144)

**Review by jwmoore1 — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]
Tested in Probo as azadmin experimenting with different content types and views. I did not notice any issues. Also, I confirmed that azcontentadmin can use Standard and Full HTML text formats.

**Review by tadean — APPROVED** [review]

**Pull request merged**